tests/test_rules.py

```diff
@@ -107,24 +107,29 @@ def test_min_rule_include_boundary_false(self) -> None:
         for value in [-42, -2, -1, 0, 1, 2, 3, 4]:
             self.assertEqual(1, len(rule.validate(value)[1]))
 
-    def test_datetime_iso_format_rule(self) -> None:
+    def test_datetime_iso_invalide(self) -> None:
         rule = IsDatetimeIsoFormat()
         now = datetime.now()
 
         actual, errors = rule.validate(now.isoformat())
         self.assertTrue(abs(now - actual) < timedelta(milliseconds=1))
         self.assertEqual([], errors)
 
-        for value in ['invalid', 42, '12.12.2020']:
+        for value in ['invalid', '2020-01-1', '12.12.2020']:
             self.assertEqual(1, len(rule.validate(value)[1]))
 
-    def test_datetime_iso_format_python_3_6(self) -> None:
-        now = datetime.now()
-        self.assertTrue(abs(now - datetime_from_iso_format(now.isoformat())) < timedelta(milliseconds=1))
-
-        for value in ['invalid', 42, '12.12.2020']:
-            with self.assertRaises(expected_exception=(AttributeError, ValueError)):
-                datetime_from_iso_format(value=value)
+    def test_datetime_iso_valid(self) -> None:
+        rule = IsDatetimeIsoFormat()
+        for dt_str, exp in (
+            ('2021-02-02T02:02:02.518993Z', datetime(2021, 2, 2, 2, 2, 2, 518993)),
```

Comment:
@LostInDarkMath hello.
I left support for the `Z` character

Reply:
As long as the rule works for something like this, I am fine with it :)
```
datetime.now().isoformat()
'2021-01-21T18:22:32.853711'
```